src/lib/systemjs-angular-loader.js

```diff
@@ -0,0 +1,49 @@
+var templateUrlRegex = /templateUrl\s*:(\s*['"`](.*?)['"`]\s*)/gm;
```

Comment:
I'd prefer that if you don't mind. It feels weird to have it here, and in general the library source shouldn't have to care about systemjs.